log_test.go

```diff
@@ -793,16 +793,16 @@ func TestCallerMarshalFunc(t *testing.T) {
 	// test custom behavior. In this case we'll take just the last directory
 	origCallerMarshalFunc := CallerMarshalFunc
 	defer func() { CallerMarshalFunc = origCallerMarshalFunc }()
-	CallerMarshalFunc = func(file string, line int) string {
+	CallerMarshalFunc = func(pc uintptr, file string, line int) string {
 		parts := strings.Split(file, "/")
 		if len(parts) > 1 {
 			return strings.Join(parts[len(parts)-2:], "/") + ":" + strconv.Itoa(line)
 		}
 
-		return file + ":" + strconv.Itoa(line)
+		return file + ":" + strconv.Itoa(line) + " " + runtime.FuncForPC(pc).Name()
```

Comment:
Shouldn't it be func:file:line?

Reply:
Yes, that's nicer. I didn't put much thought into it, as it's only used locally within the test TBH. In my personal use-case I probably won't even include the file name.